lib/fizzy/execute.cpp

```diff
@@ -254,41 +254,29 @@ inline constexpr T rotr(T lhs, T rhs) noexcept
     return (lhs >> k) | (lhs << (num_bits - k));
 }
 
-inline uint32_t clz32(uint32_t value) noexcept
+constexpr uint32_t clz32(uint32_t value) noexcept
 {
-    // NOTE: Wasm specifies this case, but C/C++ intrinsic leaves it as undefined.
-    if (value == 0)
-        return 32;
-    return static_cast<uint32_t>(__builtin_clz(value));
+    return static_cast<uint32_t>(countl_zero(value));
 }
 
-inline uint32_t ctz32(uint32_t value) noexcept
+constexpr uint32_t ctz32(uint32_t value) noexcept
```

Comment:
```suggestion
inline constexpr uint32_t ctz32(uint32_t value) noexcept
```

Reply:
https://github.com/wasmx/fizzy/pull/689#discussion_r555032201

Actually just removing these wrappers seems to be okay?

Reply:
So how about the template proposed in https://github.com/wasmx/fizzy/pull/689#discussion_r555031222 ?

Reply:
Template is fine.

Reply:
> This mean you now need to construct a unit test what will reproduce this bug.

This is still missing. It requires a special test where you pollute the stack first and then assign result of `i64.clz`. If the result is `int` some garbage should stay in the value. One case should be enough. See `TEST(execute_numeric, i64_extend_i32_u)`.

I can also handle this later.

Reply:
It looks it is not easy to omit the wrapper so I will skip creating tests for this.